content/en/docs/project-user-guide/custom-application-monitoring/visualization/overview.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Overview"
+keywords: 'monitoring, prometheus, prometheus operator'
+description: 'Overview'
+
+linkTitle: "Overview"
+weight: 2130
+---
+
+This section introduces dashboard features. You will learn how to visualize metric data on KubeSphere for your custom applications. If you do not know how to integrate your application metrics into KubeSphere monitoring system, you should read [Introduction](../../introduction) first.
+
+## Create Dashboard
+
+To create new dashboards for your application metrics, you should navigate to **Custom Monitoring** on the Project Overview page. There are free ways to create dashboards. For MySQL, Elasticsearch, and Redis, you can use built-in templates. These templates are for demonstration purposes and are updated with KubeSphere releases. Besides, you can choose to customize dashboards from scratch.
```

Comment:
There are free ways to create dashboards.
->
There are different ways to create dashboards.

Reply:
free -> three